README.md

```diff
@@ -122,6 +123,7 @@ To provide a default connection such that initial loads of the graph explorer al
     - `GRAPH_CONNECTION_URL` - `None` - See [Add a New Connection](#connections-ui)
   - Required if `USING_PROXY_SERVER=True` and `IAM=True`
     - `AWS_REGION` - `None` - See [Add a New Connection](#connections-ui)
+    - `SERVICE_TYPE` - `Neptune-db`
```

Comment:
Change this to `neptune-db` as service name is case-sensitive

Reply:
updated.